modules/gdscript/gdscript_parser.cpp

```diff
@@ -1530,7 +1530,10 @@ GDScriptParser::SuiteNode *GDScriptParser::parse_suite(const String &p_context,
 				VariableNode *variable = static_cast<VariableNode *>(statement);
 				const SuiteNode::Local &local = current_suite->get_local(variable->identifier->name);
 				if (local.type != SuiteNode::Local::UNDEFINED) {
-					push_error(vformat(R"(There is already a %s named "%s" declared in this scope.)", local.get_name(), variable->identifier->name));
+					Node *errLoc = alloc_node<PassNode>();
+					errLoc->start_line = current_suite->end_line;
+					errLoc->leftmost_column = local.start_column;
+					push_error(vformat(R"(There is already a %s named "%s" declared in this scope.)", local.get_name(), variable->identifier->name), errLoc);
```

Comment:
The variable node contains the error line and column already. What you're doing here is setting the error to the end of the current block, which might not always work. You could use the `variable` node defined earlier instead of `errLoc` to fix this bug.
Of course, I could always be wrong, and I'm more than open to have a conversation about this if so!

Reply:
ok , i was having a hard time understanding the variables but i think i get it now. Is there any way to console log a variable in godot error console?
also as @MewPurPur mentioned below, this issue seems to be fixed but the issue is still open.